public/images/svg/checkbox-unchecked.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" id="Build-Stage-Passed" x="0px" y="0px" viewBox="0 0 15 15" style="enable-background:new 0 0 15 15;" xmlns="http://www.w3.org/2000/svg">
+  <style type="text/css">
+	.st0{fill:none;stroke:#3EAAAF;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:10;}
+</style>
+  <g id="stage-passed-icon">
+    <circle class="st0" cx="7.5" cy="7.5" r="7" style="stroke: rgb(157, 157, 157);"/>
```

Comment:
The stroke style on this element could be moved into the <style> tag above, replacing the existing declaration on the `.st0` class.

Reply:
Thanks for the catch! I didn't notice it, seems like some editor have put it there 🤔And RGB format doesn't make any sense too.

Reply:
Heheh yeah absolutely. I've definitely had SVG editors do some weird things. And yeah, probably no reason to have it in RGB